package Exception;

public class E4 {
	public static void main(String[] args) {
		System.out.println("main begins");
		int a = 10;
		int b = 0;
		try {
			System.out.println("from try");
			int c = a/b;
		}
			
	}

}
